generate-offline-map/src/main/java/com/esri/arcgismaps/sample/generateofflinemap/components/MapViewModel.kt

```diff
@@ -46,6 +47,8 @@ import java.io.File
 
 class MapViewModel(private val application: Application) : AndroidViewModel(application) {
 
+    var currentSpatialReference by mutableStateOf<SpatialReference?>(null)
```

Comment:
I'm not able to understand the need for this added line. It seems like it is set via the MainScreen:
```kotlin
 onSpatialReferenceChanged = { spatialReference ->
                        mapViewModel.currentSpatialReference = spatialReference
                    }
```
But this assignment is not utilized anywhere else.